broadbean/broadbean.py

```diff
@@ -2245,6 +2245,56 @@ def rescaler(val, ampl, off):
 
         return output
 
+def joinElements(elem1: Element,
+                 elem2: Element,
+                 zero_pad: bool=True,
+                 overlap_time: float=0)->Element:
+    """
+    joins two elements by appending elem2 to elem1
+    and returns a new atomic element:
+    |-~-| |~--| |-~-~--|
+    |---|+|~--|=|---~--|
+    |---| |~--| |---~--|
+    If the channels specified in the two elements differ
+    one can choose to do zero padding
+    |-~-| |~--| |-~-~--|
+         +|~--|=|---~--|
+    |---|       |------|
+    If the channels are disjunct an overlap can be introduced
+    |-~-|       |-~--|
+          |~--| |-~--|
+         +|~--|=|-~--|
+    |---|       |----|
+    It should also be possible at some point to join elements
```

Comment:
please do not leave "feature requests" in code. we have GitHub issues for that, right?